pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -126,6 +133,30 @@ func (i *Gatherer) Gather(ctx context.Context, recorder record.Interface) error
 	)
 }
 
+// GatherPodDisruptionBudgets gathers the cluster's PodDisruptionBudgets.
+//
+// The Kubernetes api https://github.com/kubernetes/client-go/blob/v11.0.0/kubernetes/typed/policy/v1beta1/poddisruptionbudget.go#L80
+// Response see https://docs.okd.io/latest/rest_api/policy_apis/poddisruptionbudget-policy-v1beta1.html
+//
+// Location in archive: config/pdbs/
+// See: docs/insights-archive-sample/config/pdbs
+func GatherPodDisruptionBudgets(i *Gatherer) func() ([]record.Record, []error) {
+	return func() ([]record.Record, []error) {
+		pdbs, err := i.policyClient.PodDisruptionBudgets("").List(metav1.ListOptions{Limit: gatherPodDisruptionBudgetLimit})
+		if err != nil {
+			return nil, []error{err}
+		}
+		records := []record.Record{}
+		for _, pdb := range pdbs.Items {
+			records = append(records, record.Record{
+				Name: fmt.Sprintf("config/pdbs/%s", pdb.GetName()),
```

Comment:
I think we should include the corresponding namespace name same as we do in master or is that another backport? :)

Reply:
I created this for 4.5 https://github.com/openshift/insights-operator/pull/219

Reply:
Yep thanks. I think it should be OK how you did that. :)